website/sales/models/order.py

```diff
@@ -0,0 +1,260 @@
+from decimal import Decimal
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.core.validators import MinValueValidator
+from django.db import models
+from django.db.models import (
+    Sum,
+    Value,
+    F,
+    DecimalField,
+    Q,
+    IntegerField,
+    BooleanField,
+    Count,
+)
+from django.db.models.functions import Coalesce
+from django.urls import reverse
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+from queryable_properties.managers import QueryablePropertiesManager
+from queryable_properties.properties import AnnotationProperty
+
+from members.models import uuid, Member
+from payments.models import Payable, Payment
+from sales.models.product import ProductListItem
+from sales.models.shift import Shift
+
+
+def default_order_shift():
+    return Shift.objects.filter(active=True).first()
+
+
+class Order(models.Model, Payable):
+
+    objects = QueryablePropertiesManager()
+
+    class Meta:
+        verbose_name = _("order")
+        verbose_name_plural = _("orders")
+        permissions = [
+            ("custom_prices", _("Can use custom prices and discounts in orders")),
+        ]
+        ordering = ["created_at"]
+        indexes = [
+            models.Index(fields=["shift"]),
+            models.Index(fields=["id"]),
```

Comment:
I believe both of these indices are obsolete.

Reply:
I will just remove them, I trust the documentation that they are not required and consider the speedup I thought I got is due to something different